projects/bp-gallery/src/components/views/browse/SubCollections.tsx

```diff
@@ -24,7 +24,12 @@ const SubCollections = ({
     }
     return {
       name: decodeBrowsePathComponent(collection.name),
-      background: collection.thumbnail ? asApiPath(collection.thumbnail) : DEFAULT_THUMBNAIL_URL,
+      // thumbnail URL gets auto-generated by a custom strapi provider, therefore we can't check for the provider like in asUploadPath
+      background: collection.thumbnail
+        ? collection.thumbnail.startsWith('/')
```

Comment:
The `.startsWith('/')` check seems a little brittle and unreadable to me. What do the auto-generated URLs look like? Do they have a protocol like `https://` or is their only difference actually a missing leading `/`? Maybe you could add some helper like `isAutoGeneratedThumbnailURL` (maybe pick a better name :D) that could make the code more readable?

Reply:
The autogenerated URLs by the strapi s3 provider have a protocol, the normal autogenerated urls start with '/', e.g. "https://d19t2ky9si92q.cloudfront.net/58e8ff52eb97430e819064cf_098d80ad7b.png"
vs "/uploads/Badepark_30cdc7dcaa.jpg"

(CDN URL will probably be changed to cdn.harz-history.de later)

Reply:
Then maybe do something like `try { new URL(url); return true; } catch { return true }` to check for a valid url instead of the leading slash?